package.json

```diff
@@ -5,6 +5,7 @@
   "main": "bot.js",
   "scripts": {
     "start": "node bot.js",
+    "dev": "nodemon bot.js",
```

Comment:
Let's only watch 2 folders: `lib` and `skills`. See the documentation.
https://github.com/remy/nodemon#monitoring-multiple-directories

Another option would be to ignore certain directories and watch everything else.
https://github.com/remy/nodemon#ignoring-files

Reply:
Done!